src/commands/env-configs/list.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import {FlagOutput} from '@oclif/core/lib/interfaces'
+import makeDebug from 'debug'
+
+import * as MixFlags from '../../utils/flags'
+import MixCommand from '../../utils/base/mix-command'
+import {MixClient, MixResponse, MixResult} from '../../mix/types'
+import {EnvConfigListResponse, EnvConfigListTransformedData, EnvConfigsListParams} from '../../mix/api/env-configs-types'
+import {listEnvConfigs} from '../../mix/api/env-configs'
+
+const debug = makeDebug('mix:commands:env-configs:list')
+
+export default class EnvConfigsList extends MixCommand {
+  static description = `list environment configurations
+  
+  Use this command to list all environment configurations for a specific project.`
+
+  static examples = [
+    'List environment configurations for project ID 29050',
+    'mix projects:list -P 29050',
+  ]
+
+  static flags = {
+    json: MixFlags.jsonFlag,
+    ...MixFlags.tableFlags({except: ['extended', 'sort']}),
+    project: MixFlags.projectWithDefaultFlag,
+    yaml: MixFlags.yamlFlag,
+  }
+
+  async buildRequestParameters(options: Partial<FlagOutput>): Promise<EnvConfigsListParams> {
+    debug('buildRequestParameters()')
+    return {projectId: options.project}
+  }
+
+  doRequest(client: MixClient, params: EnvConfigsListParams): Promise<MixResponse> {
+    debug('doRequest()')
+    return listEnvConfigs(client, params)
+  }
+
+  get columns() {
+    return  {
+      envId: {header: 'EnvId'},
+      envName: {header: 'EnvName'},
+      envGeoID: {header: 'EnvGeoID'},
+      envGeoName: {header: 'EnvGeoName'},
+      label: {header: 'Label'},
+      value: {header: 'Value'},
+      defaultValue: {header: 'Default Value'},
```

Comment:
Use `DefaultValue` for the header value for consistency with table headers the rest of the cli.